### DISCUSSION TITLE
Separat time field

### DISCUSSION THREAD

**Discussion by Hogego** [derhansen/sf_event_mgt#966] [General]
Hi,
I would like to have a separate Timefield.
For example: I have an event for two days and want to show that each days time it's from 9 – 12 
How can I manage this?

Thanks a lot

Hogego

**Comment by derhansen** [comment]
You could extend the extension with a custom field to hold the time value. See https://github.com/derhansen/sf_event_mgt_extend_demo for an example on how to extend the main extension without modifying the original code.

Note, that a custom time field will not affect any logical constraints in regards to e.g. the calendar view or database queries, since there always the startdate and enddate fields will be considered and it is not easily possible to split the current datetime fields into logically separated fields.

**Comment by Hogego** [comment]
I have never used the extension builder before and maybe your demo version would be very helpfull for me

But this version ist for TYPO3 11 - do you have a version that works with TYPO3 10?

**Comment by derhansen** [comment]
Version for TYPO3 8.7 - 11.5 are available (see version matrix in README.md)

Downloads: https://github.com/derhansen/sf_event_mgt_extend_demo/tags
